units/Legion/Labs/legap.lua

```diff
@@ -16,7 +16,7 @@ return {
 		energystorage = 100,
 		explodeas = "largeBuildingexplosiongeneric",
 		footprintx = 6,
-		footprintz = 5,
+		footprintz = 6,
```

Comment:
not 6x9z as listed in the PR standards
`T1 Air = 6x9,`
hitbox needs adjustment as its smaller than the model